### PR TITLE
Add clyent

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

**Comment by jjhelmus** [comment]
@srossross or @dsludwig would either of you be interested in being added as maintainers of this package on conda-forge?

File: recipes/clyent/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - python
+    - setuptools
+
+test:
+  imports:
+    - clyent
+
+about:
+  home: https://github.com/Anaconda-Server/clyent
+  license: BSD
```
Comment (jakirkham) [inline recipes/clyent/meta.yaml]: Could we please specify number of clauses?

Comment (jjhelmus) [inline recipes/clyent/meta.yaml]: The project does not specify the exact license only `BSD`.  I opened an issue on GitHub asking the developers to clarify.

Comment (jakirkham) [inline recipes/clyent/meta.yaml]: Sounds good. Let's see what they say. Added a ref to the issue below.

xref: https://github.com/Anaconda-Platform/clyent/issues/11

**Comment by jjhelmus** [comment]
The feedstock for this never seems to have been created.

**Comment by jakirkham** [comment]
That's because we are backlogged again. 😞 See it is still in [`master`]( https://github.com/conda-forge/staged-recipes/tree/master/recipes/clyent ).

**Comment by jakirkham** [comment]
Added this issue ( https://github.com/conda-forge/staged-recipes/issues/923 ) to track progress.

**Pull request merged**